stdlib/xml/dom/minidom.pyi

```diff
@@ -95,74 +101,73 @@ class Node(xml.dom.Node):
         ) -> bytes: ...
 
     def hasChildNodes(self) -> bool: ...
-    def insertBefore(self, newChild, refChild): ...
+    def insertBefore(self, newChild: _N, refChild: Node) -> _N: ...
     def appendChild(self, node: _N) -> _N: ...
-    def replaceChild(self, newChild, oldChild): ...
-    def removeChild(self, oldChild): ...
+    def replaceChild(self, newChild: Node, oldChild: _N) -> _N: ...
+    def removeChild(self, oldChild: _N) -> _N: ...
     def normalize(self) -> None: ...
-    def cloneNode(self, deep): ...
-    def isSupported(self, feature, version): ...
-    def isSameNode(self, other): ...
-    def getInterface(self, feature): ...
-    def getUserData(self, key): ...
-    def setUserData(self, key, data, handler): ...
-    childNodes: Incomplete
+    def cloneNode(self, deep: bool) -> Self: ...
+    def isSupported(self, feature: str, version: str | None) -> bool: ...
+    def isSameNode(self, other: Node) -> bool: ...
+    def getInterface(self, feature: str) -> Self | None: ...
+    def getUserData(self, key: str) -> object: ...
+    def setUserData(self, key: str, data: object, handler: UserDataHandler) -> object | None: ...
```

Comment:
Returning `object` would require every use of the return value to be guarded by `isinstance` checks. `Any` is preferred for return values.
```suggestion
    def getUserData(self, key: str) -> Any: ...
    def setUserData(self, key: str, data: object, handler: UserDataHandler) -> Any: ...
```